cmd/beacond/types.go

```diff
@@ -141,25 +131,15 @@ type (
 	AvailabilityStore = dastore.Store
 
 	// BeaconBlock type aliases.
-	ConsensusBlock = consruntimetypes.ConsensusBlock[*BeaconBlock]
-	BeaconBlock    = types.BeaconBlock
-
-	// BeaconState is a type alias for the BeaconState.
-	BeaconState = statedb.StateDB[
```

Comment:
@fridrik01 I will miss the `BeaconState` name. Maybe we will re-introduce it at some point